lib/components/formik/Input.js

```diff
@@ -10,7 +10,7 @@ const FormikInput = forwardRef(({ name, ...rest }, ref) => (
       <Input
         ref={ref}
         {...field}
-        error={meta.touched && meta.error}
+        error={meta.touched ? meta.error : ""}
```

Comment:
Fix for this warning:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/85148587/176136510-87345f69-cc6d-42cd-972f-dabdd8952735.png">